src/message/__tests__/messageActionSheet-test.js

```diff
@@ -1,30 +1,31 @@
+// @flow strict-local
 import deepFreeze from 'deep-freeze';
-
+import * as eg from '../../__tests__/exampleData';
 import { constructMessageActionButtons, constructHeaderActionButtons } from '../messageActionSheet';
 
 const baseBackgroundData = deepFreeze({
-  auth: {
-    realm: '',
-    email: 'Zoe@zulip.com',
-  },
-  subscriptions: [],
-  mute: [],
+  ...eg.backgroundData,
   flags: {
+    ...eg.backgroundData.flags,
     starred: {
+      // Key has to be number according to the type declaration. $FlowFixMe.
       1: true,
+      // Key has to be number according to the type declaration. $FlowFixMe.
       2: true,
     },
   },
-  ownUser: { email: 'Zoe@zulip.com' },
 });
 
 describe('constructActionButtons', () => {
   const narrow = deepFreeze([]);
 
   test('show star message option if message is not starred', () => {
     const message = deepFreeze({
+      ...eg.streamMessage(),
       id: 3,
       reactions: [],
+      avatar_url: '',
+      client: '',
```

Comment:
Since you're using `eg.streamMessage()` now, you can just do `const message = eg.streamMessage({ id: 3 })` here (and similarly elsewhere) – there's no need to explicitly specify any properties which aren't relevant to the test.